ci/ray_ci/test_ray_docker_container.py

```diff
@@ -83,29 +234,33 @@ def test_canonical_tag(self) -> None:
     def test_get_image_tags(self) -> None:
         # bulk logic of _get_image_tags is tested in its callers (get_image_name and
         # get_canonical_tag), so we only test the basic cases here
+        sha = os.environ["BUILDKITE_COMMIT"][:6]
```

Comment:
could you replace this to `abc123`, so that it is not reading from the env?

in general, we do not want to run the same logic again in tests, but want to check the result.

like you do not want to test:

````
a = 2; b = 3
result = sum(a, b)
assert(sum(a, b) == result)
````

but you want to test:

````
a = 3; b = 3; want_sum = 5
assert(sum(a, b) == want_sum)
````

also in other places in this PR. try not to just regenerate what the code logic is doing in tests once again, but explicitly state the result you want.

Reply:
Done! I have also updated other places in the test file to lock in the expected result